composer.json

```diff
@@ -31,7 +31,7 @@
 	"require": {
 		"php": ">=5.3.0",
 		"data-values/data-values": "~1.0|~0.1",
-		"data-values/interfaces": "~0.1",
+		"data-values/interfaces": "~0.1.5",
```

Comment:
I did not expected the `~` operator to change it's meaning (from major to minor) depending on the length of the string.

Since we are still in a 0.x.x range, a 0.1 step is major and 0.0.1 is minor. So the original `~0.1.5` with the meaning `>=0.1.5 <0.2` was correct.

Now the `^0.1.5` means `>=0.1.5 <1.0".

In the end I don't care much since this is like forecasting the future.